sdk/src/main/java/io/opentelemetry/sdk/metrics/data/MetricData.java

```diff
@@ -152,6 +157,142 @@ public static DoublePoint create(
     }
   }
 
+  /**
+   * LongSummaryPoint is a single data point that summarizes the values in a time series of long
+   * values.
+   */
+  @Immutable
+  @AutoValue
+  public abstract static class LongSummaryPoint extends Point {
```

Comment:
Do we need both Long and Double Summary? In the proto we define only one (inspired by the OpenMetrics), because they believe that double is good enough to represent the sum/value at percentile in this case even for long recorded values, I would keep the aggregations separated but the export value can be only what is currently DoubleSummaryPoint.

Reply:
I'm ok with just going with double, if we think that's the right way to go. I'll make that change.

Reply:
updated to only use Doubles for the summary points.